src/background/connections/dAppConnection/models.ts

```diff
@@ -34,6 +34,7 @@ export enum DAppProviderRequest {
   AVALANCHE_SET_DEVELOPER_MODE = 'avalanche_setDeveloperMode',
   ACCOUNT_SELECT = 'avalanche_selectAccount',
   ACCOUNT_RENAME = 'avalanche_renameAccount',
+  ACCOUNTS_DELETE = 'avalanche_deleteAccounts',
```

Comment:
Should we start prefixing these methods with `wallet_` instead of `avalanche_`?
My reasoning is that this and some other methods are wallet related while `avalanche_` implies network specificity like `eth_` does

Reply:
Oh I see! Yes. I will update it to be wallet_deleteAccounts 😄 
The other ones we should change to is selectAccount and renameAccount?

Reply:
Talked to Michal and Gergely, the name change from avalanche_ to wallet_ is going to happen later on a separate ticket